docs/api.rst

```diff
@@ -4427,6 +4448,25 @@ Slash Options
       - Attachment
       -
 
+.. list-table:: Advanced Annotations
+    :widths: 25 75
+    :header-rows: 1
+
+    * - Typing Annotation
+      - Notes
+    * - :class:`Optional` [<type>]
+      - Makes the slash option not required with the type of the given <type>.
+    * - :class:`Literal` [<choice1>, <choice2>, ...]
+      - Makes the slash option into a choice of the given type. All choices have to have the same data type.
```

Comment:
have to be

Reply:
have to be what? /j

Reply:
All choices have to be the same data type.